### PR TITLE
Import color control cluster

### LINKED ISSUE #2053: Import ../third_party/connectedhomeip/src/app/clusters/color-control-server
#### Problem
<what's wrong or missing, please include any applicable:
 * expected behavior
 * actual behavior
 * steps to reproduce
 * system configuration
 * screenshots, images, logs, etc.
 >

 #### Proposed Solution
<suggested fix, suggested enhancement>

### PR DESCRIPTION
**Pull request by bhaskar-apple**
#### Problem
Import color control cluster

 #### Summary of Changes
Make changes to compile and link

fixes #2053

### CODE REVIEW

**Review by saurabhst — APPROVED** [review]

File: src/app/util/types_stub.h [hunk 1]
```diff
@@ -1928,4 +1928,10 @@ typedef struct
 #define MILLISECOND_TICKS_PER_SECOND 1000
 #define MILLISECOND_TICKS_PER_DECISECOND (MILLISECOND_TICKS_PER_SECOND / 10)
 
+#define emberEventControlSetDelayMS(control, delay) (void) 0
```
Comment (bhaskar-apple) [inline src/app/util/types_stub.h:1931]: Work to fix these tracked here
https://github.com/project-chip/connectedhomeip/issues/2431

**Review by rwalker-apple — APPROVED** [review]

**Review by bzbarsky-apple — APPROVED** [review]

**Comment by bhaskar-apple** [comment]
@BroderickCarlin @mspang @hawk248 - adding more ppl for approval

File: src/app/util/af-types.h [hunk 2]
```diff
@@ -1977,6 +1977,8 @@ typedef struct
 #define EMBER_AF_PERMIT_JOIN_FOREVER 0xFF
 #define EMBER_AF_PERMIT_JOIN_MAX_TIMEOUT 0xFE
 
+#define MAX_INT16U_VALUE (0xFFFF)
```
Comment (mspang) [inline src/app/util/af-types.h:1980]: UINT16_MAX?

**Review by mspang — APPROVED** [review]

File: src/app/util/af-types.h (hunk 2 above)
Comment (bhaskar-apple) [reply src/app/util/af-types.h:1980]: Copying over what's in the silabs SDK for now.

Comment (mspang) [reply src/app/util/af-types.h:1980]: ack

**Pull request merged**